lib/logging/Logger.js

```diff
@@ -130,4 +130,4 @@ class WebpackLogger {
 	}
 }
 
-exports.Logger = WebpackLogger;
+module.exports = Logger;
```

Comment:
Why was this changed? This breaks the `exports.LogType = ...` above

Reply:
see https://github.com/webpack/webpack/pull/9934

Reply:
Yes, I will revert